features/system-color.yml

```diff
@@ -0,0 +1,10 @@
+name: system-color
+description: The `system-color` CSS keywords allow authors to match default colors from the user agent.
+spec: https://drafts.csswg.org/css-color-4/#css-system-colors
+group: css
+status:
+  compute_from: css.types.color.system-color
+compat_features:
+  - css.types.color.system-color
```

Comment:
This is not a problem here, but CSS types in browser compat data (BCD) are quite messy. We'll probably see some cases where we'll have to write support statuses by hand, or have no status at all. I was about to suggest having a note about this, but I think feature grouping will probably do the work here.